blog.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Est Laborum - Blog</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+
+ <!--        <link rel="stylesheet" href="css/normalize.css">
```

Comment:
Since we're using version control there is no reason to commit commented out code. It's clear and a better practice.